affine/__init__.py

```diff
@@ -32,10 +32,9 @@
 # EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #############################################################################
 
-from collections import namedtuple
 import math
 import warnings
-
+from collections import namedtuple
```

Comment:
@mwtoews Can you use `force_sort_within_sections` https://pycqa.github.io/isort/docs/configuration/options.html#force-sort-within-sections to restore the old sorting? I prefer collections imports to come before math.

Reply:
Ruff supports this option [here](https://docs.astral.sh/ruff/settings/#lint_isort_force-sort-within-sections)